requirements/base.txt

```diff
@@ -398,6 +408,10 @@ psycopg2==2.9.9
     # via -r requirements/base.in
 pycparser==2.20
     # via cffi
+pydantic==1.10.14
+    # via
+    #   django-two-factor-auth
+    #   webauthn
```

Comment:
if you bump to django-two-factor-auth 1.16.0 you don't pull in this dependency

Reply:
We're pulling `django-two-factor-auth` through `maykin-2fa`, which doesn't have a pinned version of the former. Why does this require manual upgrade?

Reply:
`./bin/compile_dependencies.sh -P django-two-factor-auth` is the mechanism to bump versions of transitive dependencies

It is not required, but recommended since it uses a more modern version of webauthn + doesn't require `pydantic` which is a common source of dependency resolution problems.

Reply:
Didn't think of handling transitive deps separately.